quickstep/src/com/android/quickstep/views/TaskMenuViewWithArrow.kt

```diff
@@ -47,15 +47,15 @@ class TaskMenuViewWithArrow<T : BaseDraggingActivity> : ArrowPopup<T> {
 
         fun showForTask(
             taskContainer: TaskIdAttributeContainer,
-            alignSecondRow: Boolean = false
```

Comment:
Any files outside the `lawnchair/` directory should be ignored and kept as-is, to prevent further complications when merging Launcher3 changes.

Reply:
Good catch, missed these, I'll check again.

Reply:
15a7e5a52ceafb4ee0c45e66dd741f30974b8606